pkg/machines/libvirt.es6

```diff
@@ -170,29 +175,47 @@ LIBVIRT_PROVIDER = {
         };
     },
 
+    INIT_DATA_RETRIEVAL () {
+        logDebug(`${this.name}.INIT_DATA_RETRIEVAL():`);
+        return dispatch => {
+            dispatch(getOsInfoList());
+            return cockpit.script(getLibvirtServiceNameScript, null, { err: "message", environ: ['LC_ALL=en_US.UTF-8'] })
+                .then(serviceName => {
+                    const match = serviceName.match(/(.+)\n$/);
```

Comment:
Change this to get the first word only, this saves the `head` and `cut`  processing in shell, and drop the unnecessary helper function:
```js
.then(output => {
    const match = output.match(/(\w+)/);
    const serviceName = match ? match[0] : null;
    if (serviceName) {
        ... get libvirt state
    } else {
         console.error (...);
         // handle faillure
   }
);
```
    
```